pkg/flowaggregator/flowaggregator.go

```diff
@@ -166,11 +170,13 @@ func NewFlowAggregator(externalFlowCollectorAddr string, externalFlowCollectorPr
 		nil,
 		0,
 		registry,
+		flowAggregatorDNSName,
+		k8sClient,
 	}
 	return fa
 }
 
-func genObservationID() (uint32, error) {
+func (fa *flowAggregator) genObservationID() (uint32, error) {
 	// TODO: Change to use cluster UUID to generate observation ID once it's available
 	h := fnv.New32()
 	h.Write([]byte(flowAggregatorDNSName))
```

Comment:
shouldn't we use fa.flowAggregatorDNSName here and remove the const with same name? We seem to take the name through configMap and already use default in the cmd/flow-aggregator/options.go. Do we need another default?